11.0/Dockerfile

```diff
@@ -0,0 +1,131 @@
+FROM debian:stretch
+MAINTAINER Camptocamp
+
+# create the working directory and a place to set the logs (if wanted)
+RUN mkdir -p /opt/odoo /var/log/odoo
+
+WORKDIR "/opt/odoo"
+
+COPY ./base_requirements.txt ./
+
+# Moved because there was a bug while installing `odoo-autodiscover`. There is
+# an accent in the contributor name
+ENV LANG=C.UTF-8 \
+    LC_ALL=C.UTF-8
+
+# Install some deps, lessc and less-plugin-clean-css, and wkhtmltopdf
+RUN set -x; \
+        sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt/ stretch-pgdg main" > /etc/apt/sources.list.d/pgdg.list' \
+        && apt-get update \
+        && apt-get purge python2.7 python2.7-minimal \
+        && apt-get install -y --no-install-recommends \
+            antiword \
+            ca-certificates \
+            curl \
+            ghostscript \
+            graphviz \
+            gnupg2 \
+            less \
+            nano \
+            node-clean-css \
+            node-less \
+            poppler-utils \
+            python-libxslt1 \
+            python3-pip \
+            python3-setuptools \
+            python3-renderpm \
+            xfonts-75dpi \
+            xfonts-base \
+            tcl expect \
+            # Added only for testing, can be removed later
+            git \
+            ssh \
+            dirmngr \
```

Comment:
Here I push a reminder to remove this.

Reply:
@yvaucher We don't need it anymore?

Reply:
git and ssh where only needed to help developing the new version.
dirmngr was finally kept because it is needed with gnupg

Reply:
do we need gnupg in the image?